pyproject.toml

```diff
@@ -1,9 +1,40 @@
 [build-system]
-requires = ["setuptools", "setuptools-scm", "wheel"]
+requires = ["setuptools>=65.0", "setuptools-scm>=7.0"]
 build-backend = "setuptools.build_meta"
 
 [tool.setuptools_scm]
-write_to = "src/fmu/sumo/sim2sumo/version.py"
+
+[tool.isort]
+profile = "black"
 
 [tool.black]
 line-length = 79
+
+[project]
+name = "fmu-sumo-sim2sumo"
+requires-python = ">=3.8"
+dynamic = ["version"]
+dependencies = [
+    "sumo-wrapper-python>=1.0.3",
+    "fmu-sumo-uploader",
```

Comment:
need `@ git+https://github.com/equinor/fmu-sumo-uploader.git@main` for `pip install .` to work

Reply:
Ok, I guess it is the right thing to do for the sake of installs, but when you want to develop, it is actually in the way. you have to reinstall with the -e flag afterwards for everything to work as intended.